docs/cpp/source/installing.rst

```diff
@@ -154,6 +154,71 @@ should now merrily print the tensor (exact output subject to randomness):
   Also, make sure you specify the correct configuration in the ``cmake --build .``
   line above.
 
+System Requirements
+-------------------
+
+To ensure smooth installation and usage of LibTorch, please ensure your system
+meets the following requirements:
+
+1. **GLIBC Version**:
+  - GLIBC 2.29 or newer for cxx11 ABI version
+  - GLIBC 2.17 or newer for pre-cxx11 ABI version
```

Comment:
I'm a bit worried about this becoming out-of-date. @svekars do you know if there is any checklist during release to ensure something like this would be updated?

Also, is it worth mentioning gcc versions as well?

Reply:
Regarding the GCC version, I can add the GCC version needed if you think is worth it. I do not think it is specified anywhere else. @jbschlosser

Reply:
I think it'd be nice to have the gcc version too if possible :)

Reply:
Sounds good, I will work on that and then request a review again once is done :).